src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -11,24 +11,37 @@
 import seedu.address.model.Model;
 import seedu.address.model.pdf.Pdf;
 
+import java.io.File;
```

Comment:
The Import statements are in the wrong order bro. I think can try to resolve some of the checkstyle errors mentioned by Travis and push again? Will save us sometime towards the end to get the builds passing haha.